docs/language-basics/type-literals.md

```diff
@@ -62,6 +62,22 @@ two
 }
 ```
 
+## String template literal
+
+Single or multi line string literal can be interpolated using `${}`
+
+```typespec
+alias hello = "bonjour";
+alias Single = "${hello} world!";
+
+alias Multi = """
+  ${hello} 
+  world!
+  """;
+```
+
+Any valid expression can be used in the interpolation but only other literals will result in the template literal being assignable to a `valueof string`. Any other value will be dependent on the decorator/emitter receiving it to handle.
```

Comment:
I think we need an example of this handling.

Reply:
This should go in teh library doc though not here?